.circleci/src/jobs/promote_to_server.yml

```diff
@@ -3,12 +3,12 @@ docker:
 steps:
   - checkout:
       path: ~/project
-  - install_app_toolbelt
   - run:
       name: Install Curl
       command: |
         sudo apt update
         sudo apt install curl -y
+  - install_app_toolbelt
```

Comment:
Npm / node is not installed in the previous step, only curl

Reply:
monday morning... :D

Reply:
updated